src/api/hooks/useGetDelegationNodeInfo.ts

```diff
@@ -21,27 +30,46 @@ export function useGetDelegationNodeInfo({
   validator,
 }: DelegationNodeInfoProps): DelegationNodeInfoResponse {
   const [state, _] = useGlobalState();
-  const [commission, setCommission] = useState<Types.MoveValue>();
-  const client = new AptosClient(state.network_value);
   const {totalVotingPower} = useGetValidatorSet();
+  const {data, isLoading} = useGetAccountResources(validatorAddress);
+  const client = new AptosClient(state.network_value);
+  const [commission, setCommission] = useState<Types.MoveValue>();
+  const [delegatedStakeAmount, setDelegatedStakeAmount] =
+    useState<Types.MoveValue>();
   const [networkPercentage, setNetworkPercentage] = useState<string>();
 
   useMemo(() => {
-    const fetchData = async () => {
-      setCommission(await getValidatorCommission(client, validatorAddress));
-    };
-    fetchData();
-    setNetworkPercentage(
-      (
-        (parseInt(validator.voting_power) / parseInt(totalVotingPower!)) *
-        100
-      ).toFixed(2),
-    );
-  }, [state.network_value, totalVotingPower]);
+    if (!isLoading) {
+      const fetchData = async () => {
+        setCommission(await getValidatorCommission(client, validatorAddress));
+      };
+      fetchData();
+      setNetworkPercentage(
+        (
+          (parseInt(validator.voting_power) / parseInt(totalVotingPower!)) *
+          100
+        ).toFixed(2),
+      );
+
+      const delegationPool = data?.find(
+        (resource) =>
+          resource.type ===
+          `${DELEGATION_POOL_ADDRESS}::delegation_pool::DelegationPool`,
+      );
+
+      const delegationPoolData: DelegationPool | undefined =
+        delegationPool?.data as DelegationPool;
+
+      setDelegatedStakeAmount(delegationPoolData?.active_shares?.total_coins);
```

Comment:
looks like you set the [`total_coins` type as `string`](https://github.com/aptos-labs/explorer/pull/363/files#diff-ae78229edfd40cc88cd35326bd4731041f41fde373b63acacb9af87949c8526aR12), but set it here in a [state of type `Types.MoveValue`](https://github.com/aptos-labs/explorer/pull/363/files#diff-ae78229edfd40cc88cd35326bd4731041f41fde373b63acacb9af87949c8526aR38)